src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js

```diff
@@ -86,8 +86,11 @@ class ActivityFeed extends React.Component<Props, State> {
         if (!activeFeedEntryId) {
             return;
         }
-
-        scrollIntoView(activeFeedItemRef);
+        if (activeFeedItemRef !== null) {
+            scrollIntoView(activeFeedItemRef);
+        } else {
+            this.resetFeedScroll();
```

Comment:
This is doing the opposite of what its parent method name implies. Should this condition be handled by the calling method?